engine/install.go

```diff
@@ -126,6 +131,7 @@ poller:
 		Str("image", i.image).
 		Msg("create agent container")
 
+	i.volumes = append(i.volumes, "/var/run/docker.sock:/var/run/docker.sock")
```

Comment:
this is not thread safe. Instead of mutating `i.volumes` a local variable should be used:

```
var volumes = append(i.volumes, "/var/run/.....")
```

Reply:
Thanks for pointing this out - just pushed a fix